Added images

ShiroTohu opened the pull request:
### Added Images
It has been a while. I have added ONLY the images this time without changing the content. I had to create another script since I forgot how the other one worked haha.

The images are pulled from this steam API. I believe this is an official API judging by the domain name but I couldn't find any documentation on it.

```
https://store.steampowered.com/api/appdetails?appids={APPID}
```

fixes the issues found in pull request #43.
closes #34.

ShiroTohu committed:
Update README.md

Added only images without changing content.

rammba reviewed (commented):
Hey @ShiroTohu, thanks for the changes. I see you got contributor role in the meantime 🚀 I think all changes are great, just revert added games. You can leave images for them though.

ShiroTohu committed:
Fixed issue where removed games where added back

When adding the images it appends the HTML to the end of a given
"section" these sections where defined using the ## tag which were
present in the removed games.

3 Games were affected by this.

The error was corrected manually.
